URA-695/correlation_plots.py

```
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct  9 12:09:14 2024
+
+@author: arun
+"""
+import os
+import pandas as pd
+import re
```

Ruff (0.8.0)

`re` imported but unused

Remove unused import: `re`

(F401)